styles/styles.css

```diff
@@ -110,6 +133,36 @@
   src: local('Arial');
 }
 
+@media (width >= 600px) {
```

Comment:
actually, I didn't realize we already had a 768 breakpoint. In that case, we can use 768, so just update the 768 as needed and remove the new 600 one you created.